plugins/Wallet/js/transactions.js

```diff
@@ -33,8 +33,8 @@ function checkCriteria(txn) {
 	// Failing one check returns false but passing all checks returns true
 	return (criteria.endTime   === null || txn.confirmationtimestamp <= criteria.endTime)   &&
 	       (criteria.startTime === null || txn.confirmationtimestamp >= criteria.startTime) &&
-	       (criteria.maxValue  === null || txn.value                 <= criteria.maxValue)  &&
-	       (criteria.minValue  === null || txn.value                 >= criteria.minValue)  &&
+	       (criteria.maxValue  === null || Math.abs(txn.value)       <= criteria.maxValue)  &&
+	       (criteria.minValue  === null || Math.abs(txn.value)       >= criteria.minValue)  &&
```

Comment:
These lines also depend on what type of value is returned from `convertSiacoin()` and set per transaction object.